src/App.tsx

```diff
@@ -16,35 +19,69 @@
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
+  const [isReversed, setIsReversed] = useState(false);
+
+  const handleReset = () => {
+    setSortField(SortType.DEFAULT);
+    setIsReversed(false);
+  };
+
+  const sortedGoods = getSortedGoods(goodsFromServer, {
+    sortField,
+    isReversed,
+  });
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        <button
```

Comment:
The active button should be marked to explain to the user what action is active. It seems you have partially implemented this with the 'is-light' class, but ensure that the active state is clear and understandable for the user. [UX]